cpp/ps0003/cpp_ps0003.cpp

```diff
@@ -1,30 +1,17 @@
-/* help your brother check if the number is even or odd
-    - if the number is even, print "Number is even"
-    - if the number is odd, print "Number is odd"
-    
-    - ex:
-      => 909
-      <= Number is odd
-
-    - another ex:
-      => 20000
-      <= Number is odd
-*/
-
 #include <iostream>
 #include <string>
 
 
 int main() {
-  // variable to get the number from the user---
+  // variable to get the number from the user--- 2/2
   int number {};
 
 
   std::cout << "Enter the number : " << std::endl;
   std::cin >> number;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number